### DISCUSSION TITLE
How to use the "homotopy" function? Are there any visual examples to help to understand?

### DISCUSSION THREAD

**Discussion by Oh-mymy** [OpenModelica/OpenModelica#11070] [Q&A]
hello there：

Recently, I'm having a problem with model initialization, I looked up the _language specification_, and the "homotopy" function is mentioned. There are relevant explanations in the specification, but I didn't fully understand. So I wonder are there any visual examples to help me understand how this function works?

Thank you very much for any help.

Mark

**Comment by phannebohm** [comment]
[This section](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/solving.html?highlight=homotopy#homotopy-method) of the OM User's Guide has some useful references. There are free pdf versions online.

**Reply by Oh-mymy** [reply]
Thank you for the response.  ^_^

**Comment by casella** [comment]
You may also look up Michael Sielemann's [PhD dissertation](https://www.semanticscholar.org/paper/Device-oriented-modeling-and-simulation-in-aircraft-Sielemann/6a960aabaae9a242f363d2677cf65b4cb652ec44). He did a lot of work on homotopy in the early 2010's

**Reply by Oh-mymy** [reply]
Thank you for the response, it helped me a lot. ^_^

**Reply by phannebohm** [reply]
Nice dissertation. Perhaps you could link to it in the user's guide?

**Reply by casella** [reply]
See #11169
